src/components/Contentful/Hero.js

```diff
@@ -47,7 +48,7 @@ export default function ContentfulHero({
         <div className="row">
           <div className="col-7">
             {sectionNameComponent}
-            <h1 className="contentful-hero__page-title">{pageTitle}</h1>
+            <h1 className="contentful-hero__page-title" dangerouslySetInnerHTML={{__html: threeSpaceToLineBreak(pageTitle)}} />
```

Comment:
In order to not lose the safety of React, you could use something like this instead: https://github.com/iansinnott/react-string-replace

Reply:
ok cool